autosklearn/ensembles/ensemble_selection.py

```diff
@@ -356,3 +359,32 @@ def get_selected_model_identifiers(self) -> List[Tuple[int, int, float]]:
 
     def get_validation_performance(self) -> float:
         return self.trajectory_[-1]
+
+    def to_sklearn(self, models: Dict[str, Pipeline]) -> _BaseVoting:
+        if self.task_type in CLASSIFICATION_TASKS:
+            voter = VotingClassifier(estimators=None)
+        elif self.task_type in REGRESSION_TASKS:
+            voter = VotingRegressor(estimators=None)
+        else:
+            raise NotImplementedError(self.task_type)
+
+        weights = []
+        models_ = []
+
+        for weight, identifier in zip(list(self.weights_), list(self.identifiers_)):
+            if weight == 0.0:
+                continue
+            weights.append(weight)
+
+            sklearn_model = models[identifier].to_sklearn()  # type: ignore
+            models_.append(sklearn_model)
+
+        voter.estimators = models_
+        voter.estimators_ = models_
+        voter.weights = weights
+
+        # TODO check where to put the label encoder???
+        # voter.le_ = sklearn.preprocessing.LabelEncoder().fit(y_train)
+        # voter.classes_ = askl_ensemble.le_.classes_
```

Comment:
We also then need to make sure the encodings are the same as if they were to be refit.